### PR TITLE
french translations.

### LINKED ISSUE #2013: complete French translations
refs #2011

### PR DESCRIPTION
**Pull request by stopfstedt**
fixes #2013

### CODE REVIEW

File: app/locales/fr/translations.js [hunk 1]
```diff
@@ -184,7 +184,7 @@ export default {
     'invalidateTokens': 'Invalider tous les jetons',
     'invalidateTokensConfirmation': "Cette action va effacer tous les jetons de l’API. Vouz devrez générer nouveaux jetons pour toutes des vos applications.  Êtes vous sûr de vouloir le faire?",
     'isSelective': 'Est sélectif',
-    'isTrack': 'Is Track',
+    'isTrack': 'Est la voie',
```
Comment (saschaben) [inline app/locales/fr/translations.js]: this is incorrect: should be `Est une piste`

File: app/locales/fr/translations.js [hunk 2]
```diff
@@ -461,7 +461,7 @@ export default {
     'today': "Aujourd'hui",
     'tokenInfo': "Jetons de l’API sont utilisés d’y accèder les données de Ilios.  Votre jeton comme un mot de passe: quiconque l’obtenu pouvent apporter des modifications en votre nom.  Lire plus dans les documentation à <a href=\"{{apiDocsUrl}}\">{{apiDocsUrl}}</a>.",
     'tooManyResults': 'Votre recherche présente plus que les {{count}} resultats. Pour meilleurs de resultats, essayez votre recherche encore avec plus termes.',
-    'totalTime': 'Total time',
+    'totalTime': 'Temps total',
```
Comment (saschaben) [inline app/locales/fr/translations.js]: this is incorrect: should be `Heures totales`

**Comment by stopfstedt** [comment]
fixed.

**Comment by saschaben** [comment]
👍

**Pull request merged**